docs/cubeviz/notebook.rst

```diff
@@ -7,9 +7,9 @@ Using Cubeviz in a Jupyter Notebook
 To initialize an instance of the Cubeviz app in a Jupyter notebook, simply run
 the following code in a cell of the notebook::
 
-    >>> from jdaviz import Cubeviz
-    >>> cubeviz = Cubeviz()
-    >>> cubeviz.app  # doctest: +SKIP
+    from jdaviz import Cubeviz
+    cubeviz = Cubeviz()
+    cubeviz.app  # doctest: +SKIP
```

Comment:
thank you for switching the order here to have `cubeviz.app` last!

Reply:
Yes! Probably saving us from a lot of tickets saying "I followed the instructions but nothing showed up!" 😄